tests/test_content_loader.py

```diff
@@ -17,354 +17,255 @@
     import builtins
 
 
-def get_mocked_yaml_reader(mocked_content={}):
-    def read(yaml_file):
-        return yaml.load(mocked_content[yaml_file])
-    return read
-
-
-@mock.patch("dmutils.content_loader.YAMLLoader.read")
 class TestContentBuilder(unittest.TestCase):
+    def test_content_builder_init(self):
+        content = ContentBuilder([])
+
+        self.assertEqual(content.sections, [])
```

Comment:
Elsewhere in dmutils we're using py.test. Is it worth replacing these with asserts?